plugins/bindrole.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: bindrole
```

Comment:
I think we discussed before (https://github.com/kubernetes-sigs/krew-index/pull/180) that `bindrole` is not an ideal name for this plugin. As far as I understand, this plugin shows an overview over roles for a given subject. The current name suggests that the plugin is used to _change_ the roles.

The last suggestion from #180 was `role-summary`. WDYT about that?

Reply:
+1 that was also the outcome of https://github.com/Ladicle/kubectl-bindrole/issues/6

Reply:
I know `bindrole` is not an ideal name, but I'd rather not rename because it'd confuse current users. If it needs to change, I prefer the short name so I'd like to make it `sumrole`. Is that ok?

Reply:
Creative names are ok. That said, I recommend to choose a descriptive name, because many users will not run the command on a daily basis but once in a while. And it is easier to recognise a descriptive plugin name than a cryptic one.
However, in the space of acceptable plugin names, the choice is yours.

For your concrete suggestion, I would question what does it mean to "sum" a role? It's not at all what you intended, but users may come with similar questions.

Reply:
`rolesum`?

Reply:
sounds nice! I'll change it to that name.

Reply:
fix: 07a3827